.github/workflows/build-dev.yaml

```diff
@@ -4,17 +4,23 @@ on:
     branches:
       - main
       - 'release-*'
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
   pull_request:
```

Comment:
Remove `pull_request` trigger - I assume you are trying to use secrets in jobs?

Reply:
This requires a two phase PR
- https://github.com/nutanix-cloud-native/cloud-provider-nutanix/pull/167
- https://github.com/nutanix-cloud-native/cloud-provider-nutanix/pull/168